model/models.py

```diff
@@ -128,3 +128,35 @@ def forward(self, sub, rel):
 
 		score = torch.sigmoid(x)
 		return score
+
```

Comment:
Could you add some docu on top of the class.
Something along the lines of how it works, so others also understand it without all too much work

Reply:
added the documentation to the function forward

Reply:
also added some docu. at the top